docs/setup/minikube/getting-started-minikube.md

```diff
@@ -15,35 +15,27 @@ Follow this step-by-step guide to set up Nuclio on [Minikube](https://github.com
 
 Ensure that the following components are installed on your installation machine:
 
+- [Docker](https://docs.docker.com/get-docker/)
 - [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+- [Helm](https://helm.sh/docs/intro/install/)
 
 It's recommended that you use these drivers:
 
-- For **Mac OS** - [hyperkit driver](https://github.com/kubernetes/minikube/blob/master/docs/drivers.md#hyperkit-driver)
-- For **Linux** - [virtualbox](https://www.virtualbox.org/wiki/Linux_Downloads)
+- For **Mac OS** - [Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/)
+- For **Linux** - [Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/) or [KVM2](https://minikube.sigs.k8s.io/docs/drivers/kvm2/)
+- For **Windows** - [Docker](https://minikube.sigs.k8s.io/docs/drivers/docker/) or [Hyper-V](https://minikube.sigs.k8s.io/docs/drivers/hyperv/)
 
 ## Prepare Minikube
 
-**Start Minikube** as you normally would. Note that the following command also enables role-based access control (RBAC) (which is disabled by default on Minikube version 0.24.1 and later) so that you can get more comfortable working with an RBAC-enabled Kubernetes cluster:
+**Start Minikube** as you normally would. Note that the following command also enables role-based access control (RBAC) so that you can get more comfortable working with an RBAC-enabled Kubernetes cluster:
```

Comment:
Why did you remove the use of list-item (`-`) bullets for the steps in this section?

Reply:
because we dont suggest minikube 0.24.1, and the default behaviour is no longer a thing we need to mention

Reply:
I wasn't asking about the removed parentheses, I was asking about the formatting change of removing the list-item marks (`-`), here and for the next step, so that the steps are no longer displayed as list bullets.

Reply:
oh, gottcha

Reply:
what? can you point out what `-` did I remove?

Reply:
My bad, sorry; I mistook the minus sign for the removed line in the diff with a list-item bullet ... .